### PR TITLE
theme: add a first version of a neutral color schema to the theme

### PR DESCRIPTION
**Pull request by jwinding**
## Explain the changes you’ve made

Add 'neutral' as a new color schema in the theme. 

## Explain why these changes are made

So that we can style our components in a neutral theme in the same way as we do with other colors. 

## Explain your solution

Just adds a first quick version of gray scale colors to the primary and secondary color palettes. 
And just check how it looks for some components, really only the checkbox and the input; for which I think it looks okay. 

## How to test the changes?

Checkout this branch, go into the storybook and check the input story and the checkboxField story, for examples of how it looks with this new color schema. 

## Was this feature tested in the following environments?
- [x] Storybook on a iOS device/simulator.
- [] Building the Application on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.

## Anything else? (optional)

Should perhaps be checked for more components, but this is more of a forward looking thing, meant to be used anytime we need a neutral color theme.

### CODE REVIEW

File: source/components/atoms/Input/Input.tsx [hunk 1]
```diff
@@ -17,10 +17,7 @@ type InputProps = Omit<TextInputProps, 'onBlur'> & {
 const StyledTextInput = styled.TextInput<InputProps>`
   width: 100%;
   font-weight: ${({ theme }) => theme.fontWeights[0]}
-  background-color: ${props =>
-    props.colorSchema === 'neutral'
-      ? props.theme.colors.neutrals[5]
-      : props.theme.colors.complementary[props.colorSchema][2]};
+  background-color: ${props => props.theme.colors.complementary[props.colorSchema][2]};
```
Comment (dannil76) [inline source/components/atoms/Input/Input.tsx:20]: What does [2] mean?

**Review by dannil76 — APPROVED** [review]

**Comment by dannil76** [comment]
When checkbox is checked the backgrund color changes to white or transparent. Correct?

File: source/components/atoms/Input/Input.tsx (hunk 1 above)
Comment (jwinding) [reply source/components/atoms/Input/Input.tsx:20]: In the theme, the theme.colors.complementary has one array of color values per color, so we are just selecting the third value from that array.

**Comment by jwinding** [comment]
> When checkbox is checked the backgrund color changes to white or transparent. Correct?

Yes, transparent I think.

File: source/components/atoms/Input/Input.tsx (hunk 1 above)
Comment (dannil76) [reply source/components/atoms/Input/Input.tsx:20]: Hmm. I wish this was a bit more verbose in the code. Now I've to think...

**Pull request merged**